.github/workflows/macos.yml

```diff
@@ -12,5 +12,18 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+
       - name: Install some packages
         run: ./test
+
+      - name: Test installed brew
+        run: brew test ack
```

Comment:
since now `test` executable doesn't accept any arguments, it makes sence to move these `brew test` to `test` executable (checkout, it already has a `brew test` call).

Reply:
I Kinda like keeping it as a separate set of steps, given that you don't have to parse much if only one is failing, you'll see it in the step name that failed. WDYT?

Reply:
It makes sense. Then I would suggest to remove `brew test` from `test` executable.

Reply:
Done.